### PR TITLE
fix python+lettuce master scripts

### PR DESCRIPTION
**Pull request by estute**
@benpatterson @jzoldak 
found an error when trying to seed these

### CODE REVIEW

**Comment by benpatterson** [comment]
👍

File: platform/jobs/edxPlatformLettuceMaster.groovy [hunk 1]
```diff
@@ -98,7 +98,9 @@ secretMap.each { jobConfigs ->
         }
 
         parameters {
-            stringParam(stringParams.name, stringParams.default, stringParams.description)
+            stringParams.each { param ->
```
Comment (jzoldak) [inline platform/jobs/edxPlatformLettuceMaster.groovy]: @estute do we know why this was working before and then it wasn't? E.g. have any versions of jenkins/plugins changed or something?

Comment (estute) [inline platform/jobs/edxPlatformLettuceMaster.groovy]: @jz this was just a last minute fix to a commit I made earlier in the day

Comment (jzoldak) [inline platform/jobs/edxPlatformLettuceMaster.groovy]: OIC. makes sense. carry on. :)

**Pull request merged**